website/docs/documentationFile.md

```diff
@@ -0,0 +1,43 @@
+---
+title: documentationfile
+---
+
+Enable C# xmlDocumentationFile
```

Comment:
Please go through this and fix spacing, capitalization, etc.  It probably makes more sense to do "Documentation File" vs "DocumentationFile" as well.